### PR TITLE
Upload guides

### PR DESCRIPTION
**Pull request by nickdala**
Upload support guides to CAMS

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -29,7 +29,9 @@
                 "AZURE_SERVICEBUS_NAMESPACE": "",
                 "AZURE_SERVICEBUS_EMAIL_REQUEST_QUEUE_NAME": "",
                 "AZURE_SERVICEBUS_EMAIL_RESPONSE_QUEUE_NAME": "",
-                "CONTOSO_SUPPORT_GUIDE_REQUEST_SERVICE": ""
+                "CONTOSO_SUPPORT_GUIDE_REQUEST_SERVICE": "",
+                "AZURE_STORAGE_ACCOUNT_NAME": "",
+                "AZURE_STORAGE_CONTAINER_NAME": ""
```
Comment (adrianhall) [inline .vscode/launch.json:34]: Are we able to run locally?  I suspect we need service bus, so answer is likely no, but launch.json would make it seem we can use a local storage emulator.

File: apps/contoso-fiber/src/main/java/com/contoso/cams/support/SupportCaseService.java [hunk 2]
```diff
@@ -215,7 +215,7 @@ public void emailGuide(Long id, Long guideId) {
     }
 
     private void createEmailSupportCaseActivity(SupportCase supportCase, SupportGuide supportGuide) {
-        final String notes = "Emailing support guide " + supportGuide.getName();
+        final String notes = "Emailing support guide " + supportGuide.getUrl();
```
Comment (adrianhall) [inline apps/contoso-fiber/src/main/java/com/contoso/cams/support/SupportCaseService.java:218]: Is the URL the full URL?  If so, what happens when the site fails over?

File: infra/dev/terraform/servicebus.tf [hunk 3]
```diff
@@ -12,7 +12,7 @@ resource "azurerm_servicebus_namespace" "servicebus_namespace" {
 
   # Should be set to false, but we need it for Keda scaling rules
   # https://github.com/microsoft/azure-container-apps/issues/592
-  local_auth_enabled  = true
+  local_auth_enabled  = false
```
Comment (adrianhall) [inline infra/dev/terraform/servicebus.tf:15]: Did we fix the keda scaling rules issue?

**Review by adrianhall — APPROVED** [review]
Just a few (non-blocking) comments.

File: apps/contoso-fiber/pom.xml [hunk 4]
```diff
@@ -11,150 +11,155 @@
...
-	</build>
+        <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-stream-binder-servicebus</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.contoso.cams</groupId>
+            <artifactId>proto</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```
Comment (mnriem) [inline apps/contoso-fiber/pom.xml:137]: Where is this one coming from?

File: .vscode/launch.json (hunk 1 above)
Comment (nickdala) [reply .vscode/launch.json:34]: We're able to run locally as long as the infrastructure is deployed to Azure.

File: apps/contoso-fiber/pom.xml (hunk 4 above)
Comment (nickdala) [reply apps/contoso-fiber/pom.xml:137]: This is the proto module that define the messages EmailRequest and EmailResponse.

Comment (mnriem) [reply apps/contoso-fiber/pom.xml:137]: Surprised to see a direct SNAPSHOT version number here. When is this generated?

**Pull request merged**